File: jats-demo-model/src/main/java/com/github/pepe79/jats/demo/model/ModifierEntry.java
package com.github.pepe79.jats.demo.model;

public class ModifierEntry
{
	private int id;

	private Component component;

	private Uom uom;

	public Uom getUom()
	{
		return uom;
	}

	public void setUom(Uom uom)
	{
		this.uom = uom;
	}

	public int getId()
	{
		return id;
	}

	public void setId(int id)
	{
		this.id = id;
	}

	public Component getComponent()
	{
		return component;
	}

	public void setComponent(Component component)
	{
		this.component = component;
	}

}

File: jats-demo-webapp/src/main/ts/model/ModifierEntry.ts
class ModifierEntry
{
	private id : number;

	private component : Component;

	private uom : Uom;

	public getId() { return this.id; }
	public setId( id : number ) { this.id = id; }

	public getComponent() { return this.component; }
	public setComponent( component : Component ) { this.component = component; }

	public getUom() { return this.uom; }
	public setUom( uom : Uom ) { this.uom = uom; }


}
